app/src/main/res/layout/home_info_card.xml

```diff
@@ -65,7 +66,7 @@
 
         <Button
             android:id="@+id/action"
-            style="?materialButtonTextStyle"
```

Comment:
Is this correct? 🤔

Reply:
Oh we have this `<item name="materialButtonTextStyle">@style/Hedvig.Button.Text</item>` - changing back.